content/security/for-admins/enforce-sign-in/methods.md

```diff
@@ -28,7 +27,7 @@ It's available with Docker Desktop version 4.32 and later.
 
 In some cases, a system reboot may be necessary for enforcement to take effect.
 
->**Note**
```

Comment:
It looks like the alert type must be in all-caps

<img width="684" alt="image" src="https://github.com/user-attachments/assets/be39eae5-ba32-4f04-ac53-f73dd1233be3">

I see that this was fixed in Hugo v0.132.2: https://github.com/gohugoio/hugo/issues/12767

For now, let's maybe keep the uppercase, since the "spec" is somewhat ambiguous.